### PR TITLE
#72 Seeking as scrollable embed

### PR DESCRIPTION
**Pull request by ArcasCZ**

### CODE REVIEW

File: seeking/module.py [hunk 1]
```diff
@@ -29,15 +29,26 @@ async def seeking(self, ctx):
...
-            template = "#{id} | {name}, {timestamp}"
-            for item in items:
+        if not items:
+            embed = utils.discord.create_embed(
+                author=ctx.author,
+                title=_(ctx, "Seeking"),
+            )
+            embed.add_field(name="\u200b", value=_(ctx, "No items found"))
+            await ctx.send(embed=embed)
+            return
```
Comment (sinus-x) [inline seeking/module.py:41]: Hmm, we usually reply with plain message, embeds are used for actual data. Can you do
```py
if not items:
    await ctx.reply(_(ctx, "No items found."))
    return
```
?

**Review by sinus-x — CHANGES_REQUESTED** [review]
Nice, just one thing

File: seeking/module.py (hunk 1 above)
Comment (ArcasCZ) [reply seeking/module.py]: Changed.

File: fun/module.py [hunk 2]
```diff
@@ -811,7 +811,7 @@ def get_hyperpet_frames(avatar: Image.Image) -> List[Image.Image]:
 
         for i in range(6):
             img = "%02d" % (i + 1)
-            deform_hue = random.randint(0, 99) ** (i + 1) // 100**i / 100
+            deform_hue = random.randint(0, 99) ** (i + 1) // 100 ** i / 100
```
Comment (sinus-x) [inline fun/module.py:814]: Now there are two unrelated changes in this file, probably caused by mismatched black version. We should do something about synchronization of config of all repos, but I'm not sure what would be a proper solution.

**Review by sinus-x — CHANGES_REQUESTED** [review]

File: fun/module.py (hunk 2 above)
Comment (ArcasCZ) [reply fun/module.py:814]: I know, but I think without this changes Black will not be happy about the PR.

Comment (sinus-x) [reply fun/module.py:814]: You can pass `git commit --no-verify` to skip those checks.

Comment (ArcasCZ) [reply fun/module.py]: Should I add this as separate commit? 🤔

**Comment by sinus-x** [comment]
I would have preferred to keep the localization changes as separate commit, but whatever. All of our other scrollables have a limit of ten items, why did you choose fifteen?

**Comment by ArcasCZ** [comment]
In Issues you said it should be 10-15. So I put there 15. I can decrease it to 10 if you want to.
